kbase-extension/static/kbase/css/appCell.css

```diff
@@ -225,43 +144,126 @@ control with padding, and then scooting the icon back into its column.
   padding: 3px 5px;
 }
 
-.report-widget .panel-title > [data-toggle="collapse"]::before {
+.kb-app-cell .panel-title > [data-toggle=collapse]::before {
   display: inline-block;
   margin-left: 0;
   margin-right: 4px;
-  font-family: "FontAwesome";
+  font-family: FontAwesome;
   font-style: normal;
-  font-weight: normal;
+  font-weight: 400;
   font-size: 90%;
   width: 12px;
-  color: silver;
+  color: #c0c0c0;
   line-height: 1;
   vertical-align: baseline;
   content: "\f078 ";
 }
 
-.report-widget .panel-title > [data-toggle="collapse"].collapsed::before {
+.kb-app-cell .panel-title > [data-toggle=collapse].collapsed::before {
   margin-left: 2px;
   margin-right: 2px;
   content: "\f054 ";
 }
 
-.kb-app-cell .panel-title > [data-toggle="collapse"]::before {
+.kb-app-cell .advanced-parameter-showing {
+  display: block;
+}
+
+.kb-app-cell .advanced-parameter-hidden {
+  display: none;
+}
+
+.kb-app-cell [data-element=run-control-panel] {
+  width: 100%;
+}
+
+.kb-app-cell [data-element=tab-pane] {
+  border-top: 2px solid rgb(33 150 243);
+}
+
+.kb-app-cell [data-element=tab-pane] > div:empty {
+  padding: 0;
+}
+
+.kb-app-cell [data-element=tab-pane] > div {
+  padding: 4px;
+}
+
+.kb-app-row-clip-btn-addon,
+.kb-app-row-close-btn-addon {
+  background: transparent;
+  border: 0;
+}
+
+.kb-app-row-clip-btn-addon {
+  padding: 0 10px 0 0;
+  height: 100%;
+}
+
+.kb-app-row-clip-btn-addon:active::after {
+  background-color: #fff;
+  border-radius: 3px;
+  border: 1px solid #808080;
+  content: "copied!";
+  margin-top: -5px;
+  padding: 7px;
+  position: absolute;
+}
+
+.kb-input-group-wide {
+  width: 100%;
+}
+
+.kb-input-row-flex {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.cell.selected .btn-default.kb-app-row-close-btn {
+  color: #808080;
+}
+
+.kb-app-row-clip-btn,
+.kb-app-row-close-btn {
+  height: 100%;
+  margin-left: 1px;
+  border: none;
+  background-color: transparent;
+  color: #808080;
+}
+
+.kb-app-parameter-right-error-bar {
+  height: 28px;
+  background: #f00;
+}
+
+.kb-app-field-feedback,
+.kb-input-group-addon {
+  background: transparent;
+  border: 0;
+}
+
+.kb-input-group-addon {
+  padding: 0 0 0 10px;
+}
+
+.report-widget .panel-title > [data-toggle=collapse]::before {
   display: inline-block;
   margin-left: 0;
   margin-right: 4px;
-  font-family: "FontAwesome";
+  font-family: FontAwesome;
   font-style: normal;
-  font-weight: normal;
+  font-weight: 400;
   font-size: 90%;
   width: 12px;
-  color: silver;
+  color: #c0c0c0;
```

Comment:
this is one of the stylelint changes -- it converts all word colours to hex codes